ui/apps/platform/src/Containers/Vulnerabilities/WorkloadCves/EntityTypeToggleGroup.tsx

```diff
@@ -2,51 +2,55 @@ import React from 'react';
 import { ToggleGroup, ToggleGroupItem } from '@patternfly/react-core';
 
 import useURLStringUnion from 'hooks/useURLStringUnion';
+import { NonEmptyArray } from 'utils/type.utils';
 import { entityTabValues, EntityTab } from './types';
 
 type EntityTabToggleGroupProps = {
-    entityContext?: EntityTab;
+    entityTabs?: Readonly<NonEmptyArray<EntityTab>>;
     cveCount?: number;
     imageCount?: number;
     deploymentCount?: number;
 };
 
 function EntityTabToggleGroup({
-    entityContext,
+    entityTabs = entityTabValues,
     cveCount = 0,
     imageCount = 0,
     deploymentCount = 0,
 }: EntityTabToggleGroupProps) {
-    const [activeEntityTabKey, setActiveEntityTabKey] = useURLStringUnion(
-        'entityTab',
-        entityTabValues
-    );
+    const [activeEntityTabKey, setActiveEntityTabKey] = useURLStringUnion('entityTab', entityTabs);
 
     return (
         <ToggleGroup className="pf-u-pl-md pf-u-pt-md">
-            {entityContext !== 'CVE' && (
+            {entityTabs.includes('CVE') ? (
                 <ToggleGroupItem
                     text={`${cveCount} CVEs`}
                     buttonId="cves"
                     isSelected={activeEntityTabKey === 'CVE'}
                     onChange={() => setActiveEntityTabKey('CVE')}
                 />
+            ) : (
```

Comment:
A [PatternFly bug](https://github.com/patternfly/patternfly-react/issues/8877) was causing this to blow up with `&&` when the first condition evaluated to `false`, so the empty fragment is needed here.